components/Theme/Theme.tsx

```diff
@@ -1,6 +1,6 @@
 import { createContext, useContext, useEffect, useState } from "react";
 
-const ThemeContext = createContext();
+const ThemeContext = createContext(null);
```

Comment:
Add a typing for the context

<img width="458" alt="Captura de Tela 2022-09-13 às 19 10 17" src="https://user-images.githubusercontent.com/73349819/189978225-5329084c-683c-4702-9004-c49c49905d65.png">

And some small typing changes

<img width="457" alt="Captura de Tela 2022-09-13 às 19 11 22" src="https://user-images.githubusercontent.com/73349819/189979207-83598011-1f37-4b24-80c7-53d9644fffd3.png">

this way vscode will give autocomplete when using useTheme() hook

<img width="514" alt="Captura de Tela 2022-09-13 às 19 20 36" src="https://user-images.githubusercontent.com/73349819/189980227-079bef25-38ae-4e51-afca-e8471a0a9f54.png">